index.js

```diff
@@ -20,8 +23,40 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+
+    //Iteration3 
+    Recipe.insertMany(recipesData)
+      .then((results) => {
+        results.forEach((result) => {
+          console.log(`Recipe title: ${result.title}`);
+        });
+
+        //Iteration4 w/ Thomas begin
+        Recipe.findOneAndUpdate({
+            title: 'Rigatoni alla Genovese'
+          }, {
+            $set: {
+              duration: 100
+            }
+          }, {
+            new: true
+          })
+          .then(() => console.log("data modified succesfully"))
+          .catch(err => console.error(`Error: ${err}`));
+
+        //iteration 5
+        Recipe.deleteOne({
+            title: 'Carrot Cake'
+          }, {
+            new: true
+          })
+          .then(() => console.log("data deleted"))
+          .catch(err => console.error(`Error: ${err}`));
+
+          mongoose.connection.close()
```

Comment:
Due to this line, I do not see the results of update and delete in the db.
Could this method be placed somewhere where it waits for the completion of the .deleteOne method?